app/src/main/java/org/ole/planet/myplanet/ui/team/MyTeamsDetailFragment.kt

```diff
@@ -95,6 +99,7 @@ class MyTeamsDetailFragment : BaseNewsFragment() {
         v.findViewById<View>(R.id.add_message).setOnClickListener { showAddMessage() }
     }
 
+    @SuppressLint("NotifyDataSetChanged")
```

Comment:
this suppression is not necessary as we need to update to these newer ones we need to update to

1. `notifyItemInserted`: Use this when a new item is added to the dataset
2. `notifyItemRemoved`: Use this when an item is removed from the dataset
3. `notifyItemChanged`: Use this when an item in the dataset changes
4. `notifyItemMoved`: Use this when an item in the dataset is moved

Reply:
should I specifically use notifyItemInserted? or should I keep it as notifyDataSetChanged?